sdk/confidentialledger/azure-security-confidentialledger/src/test/java/com/azure/security/confidentialledger/ConfidentialLedgerClientTestBase.java

```diff
@@ -66,6 +78,49 @@ protected void beforeTest() {
         } catch (Exception ex) {
             System.out.println("Error thrown from ConfidentialLedgerClientTestBase:" + ex);
         }
+
+        String ledgerId = Configuration.getGlobalConfiguration().get("LEDGERID", "emily-java-sdk-tests");
+        Response<BinaryData> ledgerIdentityWithResponse = confidentialLedgerCertificateClient
+                .getLedgerIdentityWithResponse(ledgerId, null);
+        BinaryData identityResponse = ledgerIdentityWithResponse.getValue();
+        ObjectMapper mapper = new ObjectMapper();
+
+        JsonNode jsonNode = null;
+        try {
+            jsonNode = mapper.readTree(identityResponse.toBytes());
+        } catch (IOException ex) {
+            System.out.println("Caught exception " + ex);
```

Comment:
If this happens, we should fail the test. So, instead of catching this exception, we can just declare this test method to throw that'll cause test failure.

(do the same in other places where exceptions are caught).

Reply:
We should remove the whole `catch` block and let the exception bubble up.